app/configurator/components/ui-helpers.ts

```diff
@@ -939,6 +909,44 @@ const divergingPaletteKeys = [
   "PuOr",
 ] as DivergingPaletteType[];
 
+const sequentialPaletteKeys = [
+  "blues",
+  "greens",
+  "greys",
+  "oranges",
+  "purples",
+  "reds",
+] as SequentialPaletteType[];
+
+const interpolatorByName = {
+  BrBG: interpolateBrBG,
+  PRGn: interpolatePRGn,
+  PiYG: interpolatePiYG,
+  PuOr: interpolatePuOr,
+  blues: interpolateBlues,
+  greens: interpolateGreens,
+  greys: interpolateGreys,
+  oranges: interpolateOranges,
+  purples: interpolatePurples,
+  reds: interpolateReds,
+};
+const defaultInterpolator = interpolatorByName["oranges"];
+
+export const getColorInterpolator = (
+  palette?: SequentialPaletteType | DivergingPaletteType
+): ((t: number) => string) => {
+  const interpolator = interpolatorByName[palette!] ?? defaultInterpolator;
+  // If the palette is sequential, we artificially clamp the value not to display too
+  // white a value
+  const isSequential = palette
+    ? // @ts-ignore
+      sequentialPaletteKeys.includes(palette)
+    : false;
+  return isSequential
+    ? (n: number) => interpolator(n * 0.8 + 0.2)
+    : interpolator;
+};
```

Comment:
Was there a particular place in the app that led to this change?

Reply:
I saw a comment from @rdataflow at some point that said that the left side of the palette was too white. I am removing this commit from the PR and we can discuss it.